added card-creator component (without functions)

Starkiller1988 opened the pull request:
Finally started with the Card Creator

Includes:

- Card Image
- Field where you can type in the Name of a Card
- Level ( 1 - 12 )
- Type
- Attribute
- Attack
- Defence
- Race
- Image Upload Field
- Description Field

No functions yet. Will start with functions tomorrow

https://user-images.githubusercontent.com/90188949/162993323-54f6f87c-a0ee-44f2-b073-b4df90b3929a.mp4

Starkiller1988 committed:
added card-creator component (without functions)

Starkiller1988 committed:
fixes in CardCreator.tsx

codexbella reviewed (approved):
Ist ja kaum neuer Code und bei CSS weißt du vermutlich besser als ich, was du tust. :)

Starkiller1988 commented:
> Ist ja kaum neuer Code und bei CSS weißt du vermutlich besser als ich, was du tust. :)

Yes in the second commit I fixed a compile issue. Important is the first commit where the issue happened. There I implemented the basic structure for my Card Creator Component. :)

Starkiller1988 commented:
> Ist ja kaum neuer Code und bei CSS weißt du vermutlich besser als ich, was du tust. :)

And thanx for the review.
